core/src/main/scala-2/chisel3/SelectableCompat.scala

```diff
@@ -2,8 +2,13 @@
 
 package chisel3
 
-/** Does nothing in Scala 2; this stub trait is for maintaining
-  * compatibility with Scala 3's selectable, required to support
-  * structurally-typed Bundles in Scala 3.
+/** These traits do nothing in Scala 2 other than maintaining
+  * compatibility with Scala 3's Selectable and
+  * scala.reflect.Selectable, required to support structural selection
+  * in Scala 3.
+  *
+  * SelectableCompat => scala.reflect.Selectable
+  * Selectable => scala.Selectable
   */
+trait SelectableCompat
 trait Selectable
```

Comment:
ScalaDoc on one does nothing for the other:

```suggestion
/** Shim for Scala 3's `scala.reflect.Selectable`
  *
  * Allows Chisel to cross-compile Scala 2 and Scala 3
  * while enabling structural typing in Scala 3.
  */
trait SelectableCompat
/** Shim for Scala 3's `Selectable`
  *
  * Allows Chisel to cross-compile Scala 2 and Scala 3
  * while enabling structural typing in Scala 3.
  */
trait Selectable
```